src/main/java/bio/terra/service/snapshotbuilder/SnapshotBuilderService.java

```diff
@@ -403,6 +406,40 @@ public SnapshotAccessRequestResponse approveRequest(UUID id) {
     return snapshotRequestDao.getById(id).toApiResponse();
   }
 
+  static void validateGroupParams(SnapshotAccessRequestModel model, String memberEmail) {
```

Comment:
Since this would normally be `private` we usually add an annotation to make it clear that it has greater visibility so it can be tested directly
```suggestion
  @VisibleForTesting
  static void validateGroupParams(SnapshotAccessRequestModel model, String memberEmail) {
```